examples/darwin-framework-tool/commands/common/CHIPToolKeypair.mm

```diff
@@ -12,15 +12,14 @@
 static NSString * const kOperationalCredentialsIssuerKeypairStorage = @"ChipToolOpCredsCAKey";
 static NSString * const kOperationalCredentialsIPK = @"ChipToolOpCredsIPK";
 
-@interface CHIPToolKeypair ()
-@property (nonatomic) chip::Crypto::P256Keypair mKeyPair;
-@property (nonatomic) chip::Crypto::P256Keypair mIssuer;
-@property (nonatomic) NSData * ipk;
-@property (atomic) uint32_t mNow;
-@property (nonatomic, readonly) SecKeyRef mPublicKey;
-@end
+@implementation CHIPToolKeypair {
+    chip::Crypto::P256Keypair _mKeyPair;
+    chip::Crypto::P256Keypair _mIssuer;
+    NSData * _ipk;
+    uint32_t _mNow;
```

Comment:
mNow used to be atomic, right?

Reply:
Yes, but that only makes the getter and setter use a lock internally, but the getter and setter were not used, since we were already accessing the ivar directly anyway.